README.md

```diff
@@ -2,43 +2,35 @@
 
 Hi, thank you for taking on our challenge! We hope that you'll learn a lot, create an awesome project and join the Astrocoders team.
 
-In this test, we will evaluate the knowledge you already have about ReactJS and your ability to learn new technologies. In order to complete it, you must develop a Gmail interface that loads the last 40 (forty) tweets containing the **#mars** hashtag and enables the user to filter them using the search box.
+In this test, we will evaluate the knowledge you already have about ReactJS and your ability to learn new technologies. In order to complete it.
 
-## Please check:
+## How to start
+Fork this repo so we can start counting the time
 
-1. The interface that is in the [Layout](#layout) section (which already is simpler than the normal one, of course);
-2. The Twitter API (https://developer.twitter.com/en/docs/api-reference-index.html);
-3. The Technical Requisites for your preferred technology that are listed just below.
-
-## Technical Requisites for JavaScript:
-
-1. ReactJS with Hooks (https://reactjs.org/);
-2. Ramda (https://ramdajs.com);
-3. styled-components (https://styled-components.com);
-4. TypeScript (https://www.typescriptlang.org/);
-
-Don't
-- Use `for` or `while` loops;
-- Mutate variables.
+## Deadline
 
-## Technical Requisites for ReasonML:
+2 weeks after forking this repo.
 
-1. ReasonReact with Hooks (https://reasonml.github.io/reason-react/);
-2. BuckleScript Belt (https://bucklescript.github.io/bucklescript/api/Belt.html);
-3. reason-apollo-hooks (https://github.com/Astrocoders/reason-apollo-hooks);
-4. OneGraph (https://onegraph.com) or Prisma.
+## Summary
 
-## Plus
+Build a simple web app that gives a user an initial 1000 credits and this user can transfer theses credits to a fake user that you should also create.
+Each transfer should include a configurable fee  of transaction
+The app should also be able to display the current balance or a given user and enlist the history of incoming and outgoing transfer for the user.
 
-Open a tweet to see replies.
+## Stack and technical requisites
+For the fontrend we use GraphQL, ReasonML, Node+TypeScript and Rust sometimes around here. But you are free to use whatever you want as long as the client-server communication is built around GraphQL and Relay conventions.
 
-## Layout
+The frontend should be built in the React paradigm and follow Relay conventions.
 
-![screen shot 2019-01-10 at 15 26 53](https://user-images.githubusercontent.com/4806269/50987231-2f502980-14f0-11e9-9f3b-c52ef553a3d2.png)
+### Why
 
-## Deadline
+If we choose a closed tools stack you'll never be able to surprise us with what you are really good with.
```

Comment:
maybe change to something like `The stack of this test is opened so you are free to use part of the knowledge you already have, and show what you can do. However we charge ReactJS and GraphQL because they are our main core`